tasks/fix-cat1.yml

```diff
@@ -1,5 +1,6 @@
 ---
-- block:
+- name: "HIGH | RHEL-07-010010 | PATCH | The file permissions, ownership, and group membership of system files and commands must match the vendor values."
```

Comment:
I wouldn't put "PATCH" as part of the block title, but probably not worth changing it for this PR; only going forward.